File: stage1/problems/problem0/starter.py
import sys

def solve():
  """
  Reads input from stdin, solves the problem, and prints the output.
  """
  try:
    # --- Input reading ---
    a, b = map(int, sys.stdin.read().split())
    
    # --- Solution ---
    # The following variables are available:
    # a: int
    # b: int
    
    # TODO: Implement the solution
    
  except ValueError:
    # It's good practice to handle potential errors, but for this problem,
    # the input is guaranteed to be valid integers.
    pass

if __name__ == "__main__":
    solve()

File: stage1/problems/problem14/starter.cpp
#include <iostream>
#include <vector>
#include <string>
#include <sstream>

void solve() {
    // --- Input reading ---
    int n;
    std::cin >> n;
    std::vector<int> nums(n);
    for(int i = 0; i < n; ++i) {
        std::cin >> nums[i];
    }
    int target;
    std::cin >> target;

    // --- Solution ---
    // The following variables are available:
    // n: int
    // nums: std::vector<int>
    // target: int

    // TODO: Implement the solution
}

int main() {
    std::ios_base::sync_with_stdio(false);
    std::cin.tie(NULL);
    solve();
    return 0;
}
